eitprocessing/continuous_data/__init__.py

```diff
@@ -174,3 +177,26 @@ def locked(self) -> bool:
     def loaded(self) -> bool:
         """Return whether the data was loaded from disk, or derived from elsewhere."""
         return len(self.derived_from) == 0
+
+    def _sliced_copy(
+        self,
+        start_index: int,
+        end_index: int,
+        label: str,
+    ) -> Self:
+        # TODO: check correct implementation
+        cls = self.__class__
+        time = self.time[start_index:end_index]
+        values = self.values[start_index:end_index]
+        description = f"Slice ({start_index}-{end_index}) of <{self.description}>"
+
+        return cls(
+            label=label,
+            name=self.name,
+            unit=self.unit,
+            category=self.category,
+            description=description,
+            derived_from=[*self.derived_from, self],
+            time=time,
+            values=values,
+        )
```

Comment:
ContinuousData was still missing __len__. Added a suggestion in [slicedcopy_continuousdata_suggestions_psomhorst](https://github.com/EIT-ALIVE/eitprocessing/tree/slicedcopy_continuousdata_suggestions_psomhorst).